### ISSUE TITLE
[frontend] gettext-ing empty strings returns PO version info

### ISSUE THREAD

**Issue by JulienLie** [UCL-INGI/INGInious#899]
After creating a new task with a single subproblem with no context, some text appears at the place of the context.

INGInious v0.8.2

1. Go to a course
2. Create a new task
3. Create a new QCM subproblem
4. Give it a name
5. Add a single correct choice
6. Save the task
7. Check the task
8. See error just above the QCM

I suppose that the context should be empty

![image](https://user-images.githubusercontent.com/38136909/204802914-29663440-031f-4707-88e7-f36c90410c43.png)
![image](https://user-images.githubusercontent.com/38136909/204803034-616e7d63-d95f-413a-ab5f-1c98c6b5afea.png)

Desktop
 - OS: Ubuntu 22.04.1 LTS
 - Browser: Opera
 - Version 92.0.4561.43

When trying to reproduce the error, it seems that, when switching INGInious language to English, the bug disapear. But when my language is French, the bug appears.

**Comment by anthonygego** [comment]
TLDR: Empty strings should not be localized using `_` or `gettext`.

**Comment by AlexandreDoneux** [comment]
Closed with PR #979
